HelpSource/Classes/UnitTest.schelp

```diff
@@ -26,6 +26,19 @@ code::
 UnitTest.gui
 ::
 
+METHOD:: run
+Run all methods whose names begin with code::test_::.
+
+code::
+TestUnitTest.new.run;
```

Comment:
It's best to leave out `.new` for the sake of simplicity.

Reply:
Not just for simplicity, but for correctness :-)  As seen in one of the other commits in this PR, the `.run` instance method is private and should not be referenced publically.  Thanks, good catch.

Reply:
Strangely, I see two methods called `run` in the class definition of `UnitTest` (one near the top, and the other near the bottom). If we document `run`, we should also document `test`, no?

Reply:
strictly speaking we don't have private methods, but if the semantics demands, we just declare the intention. Usually either a comment in the code, or a method name that starts with `pr`.

Reply:
I know I'm the least experienced SC person in this conversation so am not well-placed to disagree ;-)  But I think the statement "strictly speaking we don't have private methods" is at least a little ambiguous, otherwise what is the existing `PRIVATE::` pragma for?  That's kind of a rhetorical question - I know more or less what it does in terms of its impact on documentation, and I presume it has no impact on code, i.e. does not protect semantically private methods from being invoked from outside.  But if that is not considered as giving us private methods on some level then I think we have a problem around how expectations are communicated developers.  Even if the language doesn't enforce privacy, I think it's fair to expect that documenting privacy should be a well-understood action which is mostly honoured by developers not because they have no choice, but because they understand that it's the right thing to do.  Larry Wall's famous quote about Perl's approach to privacy is very relevant here:

>“Perl doesn't have an infatuation with enforced privacy. It would prefer that you stayed out of its living room because you weren't invited, not because it has a shotgun”

I think it's more than reasonable if SuperCollider chooses the same approach.  But then I think declaring the intention via comments in the implementation puts us on shaky ground, because it requires all consumers to read the implementation in order to understand the API.  Using `pr` or `_` prefixes makes more sense, and using `PRIVATE::` even more so.